pkg/apis/deployment/v1/conditions.go

```diff
@@ -76,8 +76,6 @@ const (
 	ConditionTypeMemberVolumeUnschedulable ConditionType = "MemberVolumeUnschedulable"
 	// ConditionTypeMarkedToRemove indicates that the member is marked to be removed.
 	ConditionTypeMarkedToRemove ConditionType = "MarkedToRemove"
-	// ConditionTypeScaleDownCandidate indicates that the member will be picked in ScaleDown operaion.
```

Comment:
That might be a breaking change for some customer who is using these APIs. I thought we can't do such changes in patch release?

Reply:
True, it should not be removed - just deprecated